### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by asiisii**
## 🚨🤦🏽🚨 *Problem:*

Please describe the problem you're trying to solve
- merge conflict issue

## 🌟💡🌟 *Solution:*

How did you solve it?

- fix merge conflict 

## 👁‍👁‍ *Changes:*

Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring

## 💠✅💠 *CheckList:*

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -3,11 +3,27 @@ const restaurantsPath =
...
+			event_id: `${fetchId}`,
+		},
+	})
 }
 
 export const fetchWinnerData = async (eventID) => {
   console.log('triggered fetch winner')
   return await fetch()
 }
+
```
Comment (mcraig2342) [inline src/apiCalls.js:18]: Nice, glad we got this post working

File: cypress/integration/LandingPage_spec.js [hunk 2]
```diff
@@ -1,44 +1,44 @@
...
-			.get('.what')
-			.should('be.visible')
-			.get('.we')
-			.should('be.visible')
-			.get('.question')
-			.should('be.visible')
-			.get('.eatin')
-			.should('be.visible')
-	})
+// 	it('should be able to see all the landing page contents', () => {
```
Comment (mcraig2342) [inline cypress/integration/LandingPage_spec.js:6]: Looks like we will have to tweak these tests to work with the real data

**Review by Diana20920 — APPROVED** [review]
It's not easy when merge conflicts enter the chat, thank you for solving these 💯

**Review by mcraig2342 — APPROVED** [review]
Looks good, lets fix our mock data to fit more closely to what we are receiving from the back end

**Pull request merged**